src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -366,10 +463,20 @@ def _forward(self, model_inputs):
             # `generate` magic to create the mask automatically won't work, we basically need to help
             # it here.
             attention_mask = model_inputs.pop("attention_mask", None)
-            tokens = self.model.generate(
-                encoder_outputs=encoder(inputs, attention_mask=attention_mask),
-                attention_mask=attention_mask,
-            )
+            if "Whisper" in self.model.__class__.__name__:
+                # logits_processor = self.model._get_logits_processor()
+                logits_processor = []
+                logits_processor.append(TimeStampLogitsProcessor(5))
```

Comment:
What is `5` ?
Giving it to a variable  might help it make it more understandable.
If there's a sane default, let's use it, if not in the model config, and if it's clunky within model config a pipeline parameter will do.
It's hard to judge from current code which solution applies here.

Pushing this somehow into the models could help `Whisper` in self.model.__class__.__name__ is brittle (will do for the current PR).